modules/drivers/redshift/test/metabase/test/data/redshift.clj

```diff
@@ -192,3 +193,25 @@
    (sql-jdbc.conn/connection-details->spec driver @db-connection-details)
    {:write? true}
    delete-session-schema!))
+
+(def ^:dynamic *use-original-describe-database-impl?*
```

Comment:
Inverting this boolean to be explicit about why we'd override it would be clearer, e.g. something like `*override-describe-database-to-filter-by-db-name?*`.

Reply:
https://github.com/metabase/metabase/blob/e6fe24f3aa6f151b50af5bfd4983a1863830edde/modules/drivers/redshift/test/metabase/test/data/redshift.clj#L197-L200